src/python/src/setup.py

```diff
@@ -98,15 +104,23 @@
     'grpc.framework': 'grpc/framework',
 }
 
+_INSTALL_REQUIRES = (
+    'enum34==1.0.4',
+    'futures==2.2.0',
+    'protobuf==3.0.0a3'
+)
+
 setuptools.setup(
     name='grpcio',
     version='0.10.0a0',
     ext_modules=_EXTENSION_MODULES,
     packages=list(_PACKAGES),
     package_dir=_PACKAGE_DIRECTORIES,
-    install_requires=[
-        'enum34==1.0.4',
-        'futures==2.2.0',
-        'protobuf==3.0.0a3'
-    ]
+    install_requires=_INSTALL_REQUIRES,
+    setup_requires=(
+        'sphinx>=1.3',
+    ) + _INSTALL_REQUIRES,
```

Comment:
The reason it's done for "packages" is that setuptools.setup raises a TypeError if a non-list sequence type is passed for the value. My suggestion of passing lists constructed from the constant value was based on the assumption that a similar problem would arise for the other arguments to the call. If that's not the case, then yes, there would be no point to it.